src/Login.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import './Login.css';
+import logo from './Assets/logo.png';
+import mockUsers from '../src/mockusers';
+
+const Login = ({ onLogin }) => {
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const user = mockUsers.find(
+      (user) => user.username === username && user.password === password
+    );
+    if (user) {
+      onLogin(username, password);
+    } else {
+      alert('Invalid username or password');
+    }
+  };
+
+  return (
+    <div className="login-container">
+      <div className="logo-container">
+        <img src={logo} alt="Logo" className="logo" />
+      </div>
+      <div className="navbar">
+        <form onSubmit={handleSubmit}>
+          <input
+            type="text"
+            placeholder="Username"
+            className="username-field"
+            value={username}
+            onChange={(e) => setUsername(e.target.value)}
+          />
+          <input
+            type="password"
+            placeholder="Password"
+            className="password-field"
+            value={password}
+            onChange={(e) => setPassword(e.target.value)}
+          />
+          <button type="submit" className="login-button">
```

Comment:
1. You can remove type="submit" as this is the default type of a button within a form element. 
2. You need to redirect the user to the path of path="/". Even if there is nothing on the page at the moment. That way it will work when we connect the other components.

Reply:
refactored and removed 'type="Submit"

 I used the useNavigate hook from React Router in the Login component. If the user successfully logs in, you can navigate them to the desired path ("/login"). 

 User will be redirected starting from root path="/",